preliminaries.tex

```diff
@@ -1664,7 +1664,7 @@ \subsection{Path induction}
 
 Given a proof $p : \id{a}{b}$,
 path induction requires us to replace \emph{both} $a$ and $b$ with the same unknown element $x$; thus in order to define an element of a family
-$C$, for all pairs of elements of $A$, it suffices to define it on the diagonal.
+$C$, for all pairs of equal elements of $A$, it suffices to define it on the diagonal.
```

Comment:
It's a good question, but I think since this paragraph is just an _informal_ introduction to based path induction, we don't need to be more careful.